crates/autopilot/src/infra/solvers/dto/settle.rs

```diff
@@ -11,6 +11,8 @@ pub struct Request {
     /// Unique ID of the solution (per driver competition), to settle.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     pub solution_id: u64,
+    /// The last block number in which the solution TX can be included
+    pub submission_deadline: u64,
```

Comment:
nit: might be nice to make the fact that this is a block number (and not e.g. a unix timestamp) more obvious by including it in the name (something like `latest_block`) and avoiding people having to read the openapi docs?